src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -31,7 +31,7 @@
 [assembly: LoadableClass(TweedieLoss.Summary, typeof(TweedieLoss), typeof(TweedieLoss.Options), typeof(SignatureRegressionLoss),
     "Tweedie Loss", "TweedieLoss", "Tweedie", "Tw")]
 
-[assembly: EntryPointModule(typeof(ExpLoss.Options))]
+[assembly: EntryPointModule(typeof(ExponentialLoss.Options))]
```

Comment:
>ExponentialLoss [](start = 35, length = 15)

something tells me you need to run regenerateEntryPoints test :)

Reply:
Apparently not. I ran RegenerateEntryPointCatalog test and there was no diff in the json manifest or ep-list.tsv. Also the build and tests pass without running RegenerateEntryPointCatalog.